extensions/notification/destination/KafkaNotificationDestination.js

```diff
@@ -1,5 +1,33 @@
 const NotificationDestination = require('./NotificationDestination');
 const KafkaProducer = require('./KafkaProducer');
+const { ZenkoMetrics } = require('arsenal').metrics;
+
+const notificationSize = ZenkoMetrics.createCounter({
+    name: 'notification_queue_processor_notification_size',
+    help: 'Total size of the notifications that were sent',
+    labelNames: ['target'],
+});
+
+const deliveryLag = ZenkoMetrics.createSummary({
+    name: 'notification_queue_processor_notification_delivery_delay_sec',
+    help: 'Difference between the time a notification is sent and when it gets delivered',
+    labelNames: ['target', 'status'],
+});
+
+function onNotificationSent(target, status, messages, delay) {
+    // Update total size of notifications sent
+    const messageSize = new TextEncoder().encode(JSON.stringify(messages)).length;
+    notificationSize.inc({
+        target,
+    }, messageSize);
+    // update notification delivery delay
+    if (delay) {
```

Comment:
in the code, this is passed as "-1" : which is a weird value for a delay, and will break the "total lag" counter (must be monotically increasing!) and average computations...

so the condition should be `delay>0`